src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,7 +1,7 @@
 databaseChangeLog:
   - include:
       relativeToChangelogFile: true
-      file: db.changelog-diff-20180709-1627.yaml
+      file: db.changelog-diff-20180710-1621.yaml
```

Comment:
Rebuild liquibase DB for the current build of master + this branch and delete all your older changelogs.